doc/data/messages/t/trailing-newlines/bad.py

```diff
@@ -0,0 +1,2 @@
+print("apple")
+
```

Comment:
```suggestion
print("apple")
# +3: [trailing-newlines]

```

worked. Is it correct?

Reply:
Maybe we could add a comment like for trailing-whitespace ? Something like this maybe:
```suggestion
print("apple")

# The file end with 2 lines that are empty # +2: [trailing-newlines]

```

Reply:
```suggestion
print("apple")
:raw-html:`<br />`
```

Reply:
Is `:raw-html:`<br />` a special syntax to insert a newline?

Reply:
It's rst syntax to affect the html result that sphinx output. I did not test it but it should add a <br> which is a newline in HTML.

Reply:
@Pierre-Sassoulas 

```
`:raw-html:`<br />`
```

didn't work.

Reply:
I'm testing `literalinclude` directive:

https://www.sphinx-doc.org/ja/master/usage/restructuredtext/directives.html#directive-literalinclude

Reply:
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/17039389/172044239-873dce62-ef16-4199-8d20-1ceb949c1609.png">

Reply:
Required code change:

```diff
diff --git a/doc/exts/pylint_messages.py b/doc/exts/pylint_messages.py
index 0e5f13872..3ca0f467a 100644
--- a/doc/exts/pylint_messages.py
+++ b/doc/exts/pylint_messages.py
@@ -105,12 +105,10 @@ def _get_python_code_as_rst(code_path: Path) -> str:
     """
     if not code_path.exists():
         return ""
-    with open(code_path, encoding="utf-8") as f:
-        file_content = f.readlines()
     return f"""\
-.. code-block:: python
-
-{"".join("  " + i for i in file_content)}"""
+.. literalinclude:: /{code_path.relative_to(Path.cwd())}
+   :language: python
+"""
 
 
 def _create_placeholders(
```

Reply:
`_get_python_code_as_rst` looks like this:

```python
def _get_python_code_as_rst(code_path: Path) -> str:
    """Return the 'rst' representation of a python file or an empty string if the file
    does not exist.
    """
    if not code_path.exists():
        return ""
    return f"""\
.. literalinclude:: /{code_path.relative_to(Path.cwd())}
   :language: python
"""
```

Reply:
Neat !

Reply:
Generated rst file looks like this:

```rst
.. _trailing-newlines:

trailing-newlines / C0305
=========================

**Message emitted:**

Trailing newlines

**Description:**

*Used when there are trailing blank lines in a file.*

**Correct code:**

.. literalinclude:: /data/messages/t/trailing-newlines/good.py
   :language: python

**Problematic code:**

.. literalinclude:: /data/messages/t/trailing-newlines/bad.py
   :language: python



Created by the `format <https://github.com/PyCQA/pylint/blob/main/pylint/checkers/format.py>`__ checker.
```